test/common_methods_invocations.py

```diff
@@ -102,77 +102,83 @@ def __len__(self):
-        ('view', (), (dont_convert(()),), 'scalar_to_scalar'),
-        ('view', (), (1,), 'scalar_to_1d'),
+        ('add', (S, S, S), ((S, S, S),), '', (True,)),
+        ('add', (S, S, S), ((S, S),), 'broadcast_rhs', (True,)),
+        ('add', (S, S), ((S, S, S),), 'broadcast_lhs', (True,)),
+        ('add', (S, 1, S), ((M, S),), 'broadcast_all', (True,)),
+        ('add', (), ((),), 'scalar', (True,)),
+        ('add', (S, S, S), ((),), 'scalar_broadcast_rhs', (True,)),
+        ('add', (), ((S, S, S),), 'scalar_broadcast_lhs', (True,)),
+        ('add', (S, S, S), (3.14,), 'constant', (True,)),
+        ('add', (), (3.14,), 'scalar_constant', (True,)),
+        ('__radd__', (S, S, S), (3.14,), 'constant', (True, 'aten::add')),
+        ('__radd__', (), (3.14,), 'scalar_constant', (True, 'aten::add')),
+        ('sub', (S, S, S), ((S, S, S),), '', (True,)),
+        ('sub', (S, S, S), ((S, S),), 'broadcast_rhs', (True,)),
+        ('sub', (S, S), ((S, S, S),), 'broadcast_lhs', (True,)),
+        ('sub', (S, 1, S), ((M, S),), 'broadcast_all', (True,)),
+        ('sub', (S, S, S), ((),), 'scalar_broadcast_rhs', (True,)),
+        ('sub', (), ((S, S, S),), 'scalar_broadcast_lhs', (True,)),
+        ('sub', (S, S, S), (3.14,), 'constant', (True,)),
+        ('sub', (), (3.14,), 'scalar_constant', (True,)),
+        ('__rsub__', (S, S, S), (3.14,), 'constant', (True, 'aten::rsub')),
+        ('__rsub__', (), (3.14,), 'scalar_constant', (True, 'aten::rsub')),
+        ('mul', (S, S, S), ((S, S, S),), '', (True,)),
+        ('mul', (), ((),), 'scalar', (True,)),
+        ('mul', (S, S, S), ((S, S),), 'broadcast_rhs', (True,)),
+        ('mul', (S, S), ((S, S, S),), 'broadcast_lhs', (True,)),
+        ('mul', (S, 1, S), ((M, S),), 'broadcast_all', (True,)),
+        ('mul', (S, S, S), ((),), 'scalar_broadcast_rhs', (True,)),
+        ('mul', (), ((S, S, S),), 'scalar_broadcast_lhs', (True,)),
+        ('mul', (S, S, S), (3.14,), 'constant', (True,)),
+        ('mul', (), (3.14,), 'scalar_constant', (True,)),
+        ('__rmul__', (S, S, S), (3.14,), 'constant', (True, 'aten::mul')),
+        ('__rmul__', (), (3.14,), 'scalar_constant', (True, 'aten::mul')),
+        ('div', (S, S, S), (torch.rand(S, S, S) + 0.1,), '', (True,)),
+        ('div', (S, S, S), (torch.rand(S, S) + 0.1,), 'broadcast_rhs', (True,)),
+        ('div', (S, S), (torch.rand(S, S, S) + 0.1,), 'broadcast_lhs', (True,)),
+        ('div', (S, 1, S), (torch.rand(M, S) + 0.1,), 'broadcast_all', (True,)),
+        ('div', (), (uniform_scalar(0.1),), 'scalar', (True,)),
+        ('div', (S, S, S), (uniform_scalar(0.1),), 'scalar_broadcast_rhs', (True,)),
+        ('div', (), (uniform_scalar(0.1),), 'scalar_broadcast_lhs', (True,)),
+        ('div', torch.rand(S, S, S) + 1e-1, (3.14,), 'constant', (True,)),
+        ('__rdiv__', torch.rand(S, S, S) + 1e-1, (3.14,), 'constant', (True, {'cpu': ['aten::mul', 'aten::reciprocal'], 'cuda': 'prim::FusionGroup'})),
```

Comment:
I feel like having the autograd test check that a FusionGroup occurred might be brittle. If someone changes how fusion groups are constructed (i.e. to make a more powerful fusion group called something else), they will not expect to be breaking autodiff tests.